### ISSUE TITLE
how to share offline maps?

### ISSUE THREAD

**Issue by premier-lifeboat** [OpenTracksApp/OSMDashboard#155]
Hi
I try to re-use offline maps I already have downloaded for osmand (which are stored under /storage/emulated/0/android/data/...). Trying now to select map directory in osm dashboard offine version 2.15.0 I get a popup like 
"this folder can not be used"
Permissions I can give to osm app are set. I use a FP4 with iode3 (i.e. Android v12). Is this an issue related to OS? Do you have any hints how to overcome this issue?

**Comment by premier-lifeboat** [comment]
I tried in parallel some stuff on osmand - and found in the meantime an open issue there; thus a fix might come rather from there #14165

**Comment by dennisguse** [comment]
That won't work as the maps formats are different. OsmAND uses vector maps while OSMDashboard uses rendered tiles (vía maosforge).

But I don't know where the error comes from.

**Comment by pstorch** [comment]
Duplicate of #90

I also found an issue on OsmAnd stating they won't support any other map format than their own: https://github.com/osmandapp/OsmAnd/issues/8308

**Comment by premier-lifeboat** [comment]
thanks for swift reply - the authorization issue might come with latest Android version. But after your confirmation that both data formats will remain incompatible for now I will stick with osmand and your online version for opentracks  :-) (because so far I have no massive use for offline tracks - which might change)

**Comment by premier-lifeboat** [comment]
thanks for swift reply - the authorization issue might come with latest Android version. But after your confirmation that both data formats will remain incompatible for now I will stick with osmand and your online version for opentracks :-) (because so far I have no massive use for offline tracks - which might change)
